specs/GetSecurityDescriptorForAppContainerNames/GetSecurityDescriptorForAppContainerNames.md

```diff
@@ -61,6 +61,26 @@ STDAPI GetSecurityDescriptorForAppContainerNames(
 
 If the function succeds, the returned `SECURITY_DESCRIPTOR` must be freed by calling [LocalFree](https://docs.microsoft.com/en-us/windows/desktop/api/winbase/nf-winbase-localfree).
 
+## WinRT
+```c#
+namespace Microsoft.Security.AccessControl
+{
+  struct AppContainerAccess
+  {
+    String appContainerName;
+    UInt32 accessMask;
+  }
+
+  static runtimeclass SecurityDescriptorHelpers
+  {
+    static String GetSDDLForAppContainerNames(
```

Comment:
Strange that this generates the SDDL but the native one generates the actual `SECURITY_DESCRIPTOR`. Is there any reasonable way you can use the SDDL in a projection where you couldn't use the SD? (In other words, does this WinRT API give you a result that effectively requires you to call into Win32 anyway?)

Reply:
You could use the SDDL in places like manifests. (There's really no good way to return a `SECURITY_DESCRIPTOR` to WinRT; maybe a `byte[]`?)

Reply:
CONSIDER: Alternate `byte[] GetSecurityDescriptorBytesFromAppContainerNames` for use with .NET's `RawSecurityDescriptor` constructor. Use customer feedback to know what's important.